src/test/java/uk/gov/ons/census/fwmtadapter/messaging/RefusalReceiverTest.java

```diff
@@ -66,6 +70,33 @@ public void testRefusalMessageFromFieldChannel() {
     verifyZeroInteractions(rabbitTemplate);
   }
 
+  @Test
+  public void testRefusalNotFromFieldForEstabAddressLevelIsIgnored() {
+    // Given
+    RabbitTemplate rabbitTemplate = mock(RabbitTemplate.class);
+
+    ResponseManagementEvent event = easyRandom.nextObject(ResponseManagementEvent.class);
+    event.getEvent().setType(EventType.REFUSAL_RECEIVED);
+    event.getEvent().setChannel("CC");
+    event.getPayload().getRefusal().getCollectionCase().setId(TEST_CASE_ID);
+
+    // When
```

Comment:
🎨 Some minor comment pedentry, two `// When` comments, I think this first one is still just setting up mocks and data so still doing `// Given` type stuff.